airbyte-integrations/connectors/source-hubspot/source_hubspot/client.py

```diff
@@ -13,7 +13,9 @@
     CampaignStream,
     ContactListStream,
     ContactsListMembershipsStream,
+    CRMSearchStream,
```

Comment:
Duplicate import. Please run `./gradlew format`

Reply:
Done in 719b678.